src/App.jsx

```diff
@@ -12,6 +13,7 @@ function App() {
       <Routes>
         <Route path="/" element={<Home />} />
         <Route path="/articles/:article_id" element={<ArticleDetails />} />
+        <Route path="/topics/*" element={<Topic />} />
```

Comment:
This should be a parametric endpoint like `/topics/:topic` to be accessed effectively by `useParams` later